pages/common/mpv.md

```diff
@@ -26,7 +26,7 @@
 
 - Play a file at a specified speed (1 by default):
 
-`mpv --speed {{0.01..100}} {{path/to/file}}`
+`mpv --speed = {{0.01..100}} {{path/to/file}}`
```

Comment:
```suggestion
`mpv --speed={{0.01..100}} {{path/to/file}}`
```
I checked it on my Linux machine and it seems that there shouldn't be any space between.